src/components/Field_Input.tsx

```diff
@@ -0,0 +1,66 @@
+import Image from "next/image";
+import {
+  FieldValues,
+  useController,
+  UseControllerProps
+} from "react-hook-form";
+
+export interface FormProps<T extends FieldValues = FieldValues>
+  extends UseControllerProps<T> {
+  label?: string;
+  placeholder?: string;
+  icon?: string;
+  required?: boolean;
+}
+
+/*
+This is a component for the HTML `<input>' tag
+Input:
+  label: The label displayed on the text box
+  placeholder: The placeholder message displayed on the text box
+  icon: An optional Image path to display an .svg file
+Output:
+  A input text box that is compatible w/ React-hook-forms
+*/
+export default function Field_Input<T extends FieldValues = FieldValues>({
+  label,
+  placeholder = "",
+  icon = "",
+  ...props
+}: FormProps<T>) {
+  const { field } = useController(props);
+
+  placeholder == "" ? (placeholder = "Enter " + props.name) : "";
```

Comment:
It's generally not a good idea to mutate props. You should save it as a separate variable.